### DISCUSSION TITLE
⭐️Help me to get the Galaxy Brain badge!⭐️ I will help you too⭐️

### DISCUSSION THREAD

**Discussion by olx24** [Schweinepriester/github-profile-achievements#447] [Q&A]
Help me here.

**Comment by abhinit21** [comment]
@olx24 

Answer Here! #451 

waiting ⌚- Accept this answer 😊
Just mention me when you answer @abhinit21

Done 👍

**Comment by haithamaouati** [comment]
If my answer helped you, you could mark it as answered. Please.

**Comment by muhammadnurulahsan** [comment]
star and ill star your repo back [link it below]
https://github.com/muhammadnurulahsan/Carpenco-Store-Client-Side
https://github.com/muhammadnurulahsan/Carpenco-Store-Server-Side

**Comment by abidurrahman11** [comment]
Guys don't ever run for badges. Improve your coding skill. Make some great repo. Help other with issues. Badges will start appearing in your profile.
Good Luck😍
